pkg/insights/insightsclient/insightsclient.go

```diff
@@ -114,6 +114,15 @@ func clientTransport(authorizer Authorizer) http.RoundTripper {
 	return transport.DebugWrappers(clientTransport)
 }
 
+func getCurrentVersion(cv* configv1.ClusterVersion) string {
+	for _, history := range cv.Status.History {
+		if history.State == "Completed" {
+			return history.Version
```

Comment:
The cluster operator resoruce has `versions` field, that is probably more relevant for this (and doesn't rely on "completed" state), see https://docs.openshift.com/container-platform/4.4/rest_api/config_apis/clusteroperator-config-openshift-io-v1.html

Example from real cluster:

```
apiVersion: config.openshift.io/v1
kind: ClusterOperator
metadata:
  annotations:
    exclude.release.openshift.io/internal-openshift-hosted: 'true'
  creationTimestamp: '2020-09-06T01:27:05Z'
  generation: 1
  managedFields:
    - apiVersion: config.openshift.io/v1
      fieldsType: FieldsV1
      fieldsV1:
        'f:metadata':
          'f:annotations':
            .: {}
            'f:exclude.release.openshift.io/internal-openshift-hosted': {}
        'f:spec': {}
        'f:status': {}
      manager: cluster-version-operator
      operation: Update
      time: '2020-09-06T01:27:05Z'
    - apiVersion: config.openshift.io/v1
      fieldsType: FieldsV1
      fieldsV1:
        'f:status':
          'f:conditions': {}
          'f:extension': {}
          'f:relatedObjects': {}
          'f:versions': {}
      manager: insights-operator
      operation: Update
      time: '2020-09-06T03:04:27Z'
  name: insights
  resourceVersion: '24106'
  selfLink: /apis/config.openshift.io/v1/clusteroperators/insights
  uid: 77f6eb92-ad73-4fde-8eb8-f39edc96e033
spec: {}
status:
  conditions:
    - lastTransitionTime: '2020-09-06T03:04:11Z'
      status: 'False'
      type: Degraded
    - lastTransitionTime: '2020-09-06T03:04:11Z'
      status: 'True'
      type: Available
    - lastTransitionTime: '2020-09-06T03:04:27Z'
      message: Monitoring the cluster
      status: 'False'
      type: Progressing
    - lastTransitionTime: '2020-09-06T03:04:27Z'
      status: 'False'
      type: Disabled
  extension: {}
  relatedObjects:
    - group: ''
      name: openshift-insights
      resource: namespaces
    - group: apps
      name: insights-operator
      namespace: openshift-insights
      resource: deployments
  versions:
    - name: operator
      version: 4.5.7

```

Reply:
But which version are we interested in ? 
The version of openshift running on the cluster or the version of the insights operator or they are the same thing ?

Reply:
It's usually the same, but even if it wasn't, the version of the operator is the thing we're interested in. The reason is to be able to do some backward compatibility conditions and version of the client/insights-operator is more useful than version of the cluster.

Reply:
> The version of openshift running on the cluster or the version of the insights operator or they are the same thing ?
This seems to be true, for all base openshift operators, but it wouldn't be true about the user's own clusteroperators.

So what I'm trying to say is that getting this info from the clusteroperators would be actually more complicated(ie.: it would require more code changes).

Reply:
:+1:

Reply:
@iNecas Does it look right now ? :)